torch/fx/subgraph_rewriter.py

```diff
@@ -60,25 +63,52 @@ def attributes_are_equal(pn : Node, gn : Node) -> bool:
             return False
         match_found = all(self._match_nodes(pn_, gn_) for pn_, gn_
                           in zip(pn.all_input_nodes, gn.all_input_nodes))
+        match_found = False
+        if pn.op == "output":
+            # This covers the case in which the original graph Node that
+            # matched to the pattern subgraph's output has more than
+            # one input Node
+            for perm in list(permutations(gn.all_input_nodes)):
+                match_found = all(self._match_nodes(pn_, gn_) for pn_, gn_
+                                  in zip(pn.all_input_nodes, perm))
+                if match_found:
+                    break
```

Comment:
We should assert that the pattern `Graph` has exactly one output value and this branch goes away, right?

Reply:
I have the `break` in there to prevent us from iterating through the entire list of permutations